src/content/docs/agile-handbook/key-concepts/agile-roles.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Agile roles"
+template: basicDoc
+topics:
+  - Docs agile handbook
+---
+Like most agile teams, we divide the team into team members, scrum masters, product owners, and managers. Because a content team has different needs than an engineering team, we ask somewhat different things of each of those roles:
```

Comment:
Not a massive hole, but it might be a good idea to add a reminder about our two-squad system here. We mention it a few times in this doc and only briefly touched on it in a prior doc.

Reply:
We could just link to the section we talk about squads:  /docs/agile-handbook/key-concepts/agile-sprints-profusion-of-terms/#teams-and-squads